src/common/CException.cpp

```diff
@@ -308,6 +308,7 @@ void _cdecl Signal_Break(int sig = 0)		// signal handler attached when using sec
 
 void _cdecl Signal_Illegal_Instruction(int sig = 0)
 {
+#ifndef __APPLE__

```

Comment:
Can i ask why this security measure was disabled?

Reply:
yeah, i think cmake toolchain is no right.